src/main/asciidoc/query-by-example.adoc

```diff
@@ -56,7 +58,8 @@ The preceding example shows a simple domain object.
 You can use it to create an `Example`.
 By default, fields having `null` values are ignored, and strings are matched by using the store specific defaults.
 
-NOTE: Inclusion of properties into a Query by Example criteria is based on nullability. Properties using primitive types (`int`, `double`, …) are always included unless <<query-by-example.matchers,ignoring the property path>>.
+NOTE: Inclusion of properties into a Query by Example criteria is based on nullability.
+Properties using primitive types (`int`, `double`, …) are always included unless <<query-by-example.matchers,ignoring the property path>>.
```

Comment:
Change to: 

Properties that use primitive types (`int`, `double`, and so on) are always included unless the application<<query-by-example.matchers,ignores the property path>>.

----------------

(If `application` is incorrect, change it to whatever is correct, but something has to do the ignoring.)

Reply:
It's the `ExampleMatcher` that does the ignoring.